app/src/main/java/com/sun/moviesun/extension/ActivityExtension.kt

```diff
@@ -0,0 +1,11 @@
+package com.sun.moviesun.extension
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+
+fun AppCompatActivity.setUpToolbar(toolbar: Toolbar, title_: String = "") {
+  setSupportActionBar(toolbar)
+  supportActionBar?.run {
+    title = title_
```

Comment:
```suggestion
    this?.title = title
```
hoặc e có thể đổi lại tên title trong value method đi e.
không nên để _ như vậy